docs/resources/as_policy.md

```diff
@@ -4,17 +4,19 @@ subcategory: "Auto Scaling"
 
 # huaweicloud_as_policy
 
-Manages a AS Policy resource within HuaweiCloud.
+Manages an AS policy resource within HuaweiCloud.
```

Comment:
r Manages/Manage

Reply:
we use **Manages** in most of the resources